### DISCUSSION TITLE
(error token is "/ 100 ")

### DISCUSSION THREAD

**Discussion by frant4** [RandomNinjaAtk/docker-lidarr-extended#68] [Q&A] [answered]
Hi,
When running video script, I get this in logs 
`/config/extended/scripts/video.sh: line 131: / 100 : syntax error: operand expected (error token is "/ 100 ")`
any ideas why this is happening ?

**Comment by RandomNinjaAtk** [comment, marked as answer]
I'm making a guess, but based on your log output it looks like the script had trouble communicating with MusicBrainz website, make sure you can access that website from the machine that is running the container.

You would need to show more of the log data, but I'm guessing it did not find any recordings for whatever artist it was processing...

Function for reference: 
https://github.com/RandomNinjaAtk/docker-lidarr-extended/blob/987eea2472215ad892b1e60d6c4dff0c010e9037/root/scripts/video.sh#L94-L148

**Reply by frant4** [reply]
Your guess is right, I routed lidarr container network trough binhex qbit vpn container I tested this yesterday and it was working fine music-videos downloaded etc.
I then reinstalled lidar-extended container today and suddenly can't access MusicBrainz anymore.
I tried restarting the VPN container, then lidarr-extended, but still can't access MusicBrainz.
Should I reconfigure my VPN container to a different IP? Or is there another/easier way to fix this?

**Reply by RandomNinjaAtk** [reply]
Thats a bit out of my wheelhouse, maybe someone else can chime in to assist.
